admin-dev/themes/new-theme/js/theme.js

```diff
@@ -1,20 +1,22 @@
-import $ from 'expose?$!expose?jQuery!jquery';
-import 'jquery-ui/jquery-ui.js';
-import Tether from 'expose?Tether!tether';
-import 'bootstrap/dist/js/bootstrap.min.js';
-import 'magnific-popup/dist/jquery.magnific-popup.min.js';
-import 'PrestaKit/dist/js/select2.min.js';
-import 'PrestaKit/dist/js/bootstrap-switch.min.js';
-import 'PrestaKit/dist/js/jquery.pstagger.min.js';
-import 'PrestaKit/dist/js/prestashop-ui-kit.js';
 
+// Plugins CSS
+
+import 'dropzone/dist/min/dropzone.min.css';
```

Comment:
If you import the unminified version, build will be longer, but resulting file is likely to be smaller because working on a lot of source files gives the minifier more room to optimize